taginfo.json

```diff
@@ -0,0 +1,41 @@
+{
+    "data_format": 1,
+    "data_url": "https://raw.githubusercontent.com/Rudloff/openvegemap/master/taginfo.json",
+    "project": {
+        "name": "OpenVegeMap",
+        "description": "Find vegetarian and vegan restaurants in your city",
+        "project_url": "https://github.com/Rudloff/openvegemap",
+        "doc_url": "https://play.google.com/store/apps/details?id=pro.rudloff.openvegemap",
+        "icon_url": "https://raw.githubusercontent.com/Rudloff/openvegemap/master/img/logo_32.png"
+    },
+    "tags": [
+{ "key": "diet:vegan",  "object_types": [ "node", "area" ], "description": "Displayed in popup" },
+{ "key": "diet:vegan", "value": "only", "object_types": [ "node", "area" ], "icon_url": "https://imgplaceholder.com/64x64/72af26/ffffff/fa-dot-circle-o" },
+{ "key": "diet:vegan", "value": "yes", "object_types": [ "node", "area" ], "icon_url": "https://imgplaceholder.com/64x64/72af26/ffffff/fa-circle" },
+{ "key": "diet:vegetarian",  "object_types": [ "node", "area" ], "description": "Displayed in popup" },
+{ "key": "diet:vegetarian", "value": "only", "object_types": [ "node", "area" ], "icon_url": "https://imgplaceholder.com/64x64/728224/ffffff/fa-circle-thin" },
+{ "key": "diet:vegetarian", "value": "yes", "object_types": [ "node", "area" ], "icon_url": "https://imgplaceholder.com/64x64/728224/ffffff/fa-circle-o" },
+{ "key": "diet:vegetarian", "value": "no", "object_types": [ "node", "area" ], "icon_url": "https://imgplaceholder.com/64x64/d63e2a/ffffff/fa-ban" },
+{ "key": "shop", "value": "bakery", "object_types": [ "node", "area" ], "description": "🥖 " },
```

Comment:
I don't think adding the emoji in the description is really helpful. What do you think of using something like "Displayed as an emoji in tooltip"?

Reply:
If I can beg you to reconsider, a lot of projects use icons; see [emergency=fire_hydrant](https://taginfo.openstreetmap.org/tags/emergency=fire_hydrant#projects) for an example of several different interpretations. [iD Editor](https://taginfo.openstreetmap.org/projects/id_editor#tags) and [OsmAnd](https://taginfo.openstreetmap.org/projects/osmand#tags) probably have over a thousand, and there many thousands of [Traffic Signs](https://taginfo.openstreetmap.org/projects/traffic_signs_eur#tags).

So I offer three choices:

1. "Displayed as an emoji in tooltip" (your suggestion)
2. "🥖 Displayed as an emoji in tooltip" (harmonizes the entries with others that start with an image
3. "Displayed as 🥖 in tooltip" (uses the emoji as an easily parsed (possibly screen-read) glyph rather than an image blob)

Thoughts?

(Note: I thought I saved a comment with something like this before, but a refresh isn't displaying it.)

Reply:
I didn't know this was a thing. I guess 2 could be a good compromise then.

Reply:
Done. Ready for merging.